### ISSUE TITLE
Affichage des transactions

### ISSUE THREAD

**Issue by vincentux** [duniter/cesium#281]
Peut-on ajouter à l'affichage des transactions, un système de filtre et de tri ?
J'aimerai pouvoir filtrer les transactions reçues de telles ou telles personnes ou de Remuniter ... etc
ET/OU
pouvoir les trier (par ordre alphabétique, de la plus petite transaction à la plus grande ... etc)

Voila, je sais pas si c'est dans tes priorités et/ou si c'est trop compliqué a réaliser, mais ça aiderait à s'y retrouver dans l'affichage des transactions ...

**Comment by blavenie** [comment]
oui, c'est possible, mais sans doute via Cesium+ avec ES dernière pour avoir une API plus souple que BMA...
